x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_list_page/index.tsx

```diff
@@ -519,6 +520,17 @@ export const AgentListPage: React.FunctionComponent<{}> = () => {
           <EuiSpacer size="l" />
         </>
       )}
+      {/* TODO serverless agent soft limit */}
+      {showUnhealthyCallout && (
```

Comment:
@nchaulet Is this an accidental change? I am seeing a double notification on Fleet UI on main:

<img width="1480" alt="image" src="https://github.com/elastic/kibana/assets/90178898/f628bc55-c058-431e-8a09-bf5ba6d36c4e">